docs/BreakingChanges/091 EntityFramework 6.0 loads very slowly in apps launched from Visual Studio.md

```diff
@@ -0,0 +1,22 @@
+## 91: EntityFramework 6.0 loads very slowly in apps launched from Visual Studio
```

Comment:
{
  "Id": "91",
  "Title": "EntityFramework 6.0 loads very slowly in apps launched from Visual Studio",
  "Details": "Launching an app from Visual Studio 2013 that uses EntityFramework 6.0 can be very slow.",
  "Suggestion": "This issue is fixed in EntityFramework 6.0.2. Please update EntityFramework to avoid the performance issue.",
  "Link": "https://entityframework.codeplex.com/workitem/1749",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis